packages/core/src/browser/shell/side-panel-handler.ts

```diff
@@ -462,6 +462,7 @@ export class SidePanelHandler {
 
         if (hideDockPanel) {
             container.addClass(COLLAPSED_CLASS);
+            this.updateSashState(this.container, true);
```

Comment:
Minor: We could move this call to `updateSashState` and the one below before the `if` and use `updateSashState(this.container, hideDockPanel)`.

Reply:
@msujew Thanks for the feedback! I have pushed changes that address your comments.